package.json

```diff
@@ -20,7 +20,7 @@
   "homepage": "https://github.com/broccolijs/broccoli",
   "dependencies": {
     "broccoli-node-info": "1.1.0",
-    "broccoli-slow-trees": "^2.0.0",
+    "broccoli-slow-trees": "3.0.1",
```

Comment:
@bekzod why did you pin this dependency instead of using a caret constraint?

Reply:
because I am crazy :D .... tbh I think it was accident

Reply:
sorry